drivers/gpio/gpio_bd8lb600fs.c

```diff
@@ -0,0 +1,266 @@
+/*
+ * Copyright (c) 2022 SILA Embedded Solutions GmbH
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT rohm_bd8lb600fs
+
+/**
+ * @file Driver for BD8LB600FS SPI-based GPIO driver.
+ */
+
+#include <errno.h>
+
+#include <zephyr/kernel.h>
+#include <zephyr/device.h>
+#include <zephyr/init.h>
+#include <zephyr/drivers/gpio.h>
+#include <zephyr/drivers/gpio/gpio_utils.h>
+#include <zephyr/drivers/spi.h>
+#include <zephyr/logging/log.h>
+#include <zephyr/sys/byteorder.h>
+
+#include "gpio_bd8lb600fs.h"
+
+LOG_MODULE_REGISTER(gpio_bd8lb600fs, CONFIG_GPIO_LOG_LEVEL);
+
+#define OUTPUT_OFF_WITH_OPEN_LOAD_DETECTION  0b11
+#define OUTPUT_ON			     0b10
+#define WAIT_TIME_RESET_ACTIVE_IN_US	     1000
+#define WAIT_TIME_RESET_INACTIVE_TO_CS_IN_US 10
+
+/*!
```

Comment:
I would suggest removing doxygen from this file.

Reply:
done